src/testing/storage.zig

```diff
@@ -501,8 +501,8 @@ pub const Storage = struct {
         storage: *const Storage,
         copy_: u8,
     ) *const superblock.SuperBlockHeader {
-        const offset = vsr.Zone.superblock.offset(superblock.areas.header.offset(copy_));
-        const bytes = storage.memory[offset..][0..superblock.areas.header.size_max];
+        const offset = vsr.Zone.superblock.offset(superblock.SuperBlockZone.header.offset(copy_));
```

Comment:
This shouldn't compile? We renamed `offset` I think?

Reply:
That is a `vsr.Zone`; I didn't rename `vsr.Zone.offset`.

Reply:
Oh wait, I didn't see the inner one